project/src/js/ui/round/view/button.js

```diff
@@ -298,5 +298,29 @@ export default {
           () => window.plugins.toast.show(i18n('notes'), 'short', 'bottom')
         )} />
     );
+  },
+  returnToTournament: function(ctrl) {
+    function handler() {
+      ctrl.hideActions();
+      m.route('/tournament/' + ctrl.data.tournament);
+    }
+    return (
+      <button key="returnToTournament" config={helper.ontouch(handler)}>
+        <span className="fa fa-trophy" />
+        Return to Tournament
+      </button>
+    );
+  },
+  withdrawFromTournament: function(ctrl) {
```

Comment:
The game screen on the website has a "withdraw" button (http://imgur.com/Lzu5aWp). It's a nice convenience button to have so that the user doesn't have to hit the "back to tournament" button from the game page and then try to hit the "withdraw" button on the tournament page before the tournament sockets sends a redirect.